### ISSUE TITLE
susanna 

### ISSUE THREAD

**Issue by jsaintlauren** [srophe/srophe-app-data#168]
Need a new record created in Q 

http://wwwb.library.vanderbilt.edu/exist/apps/srophe/person/2092 
has 2 susannas

The one in http://syriaca.org/work/819 is from John of E 
the one of the other two texts listed in the biblical susanna.

**Comment by davidamichelson** [comment]
@jedwardwalters FYI

**Comment by wlpotter** [comment]
I don't think this was resolved? We have the following works commemorating https://syriaca.org/person/2092

- http://syriaca.org/work/819
- http://syriaca.org/work/1797
- http://syriaca.org/work/1834

The first is correct, but I'm not sure if the second and third are the Susanna in John of E or if they are for a different person?

I don't think there is a record created for the biblical Susanna?

**Comment by wlpotter** [comment]
FYI based on the descriptions given for the Mingana mss ([529](https://archive.org/details/MinganaCatalogueOfTheMinganaCollection.I/page/n491/mode/2up), item D and [598](https://archive.org/details/MinganaCatalogueOfTheMinganaCollection.I/page/n575/mode/2up), item F.) I'm pretty sure work 1797 is for the biblical Susanna. I'm not sure if we already have a work record for this? It would be part of https://syriaca.org/work/9722 right? Should this still be in BHSE?

I think work 1834 is also on the biblical Susanna based on [Mingana ms 152](https://archive.org/details/MinganaCatalogueOfTheMinganaCollection.I/page/n183/mode/2up) (item G in Mingana's catalogue). Is this a commentary on the biblical work? In which case, do we consider it hagiography?
